README.md

```diff
@@ -2,27 +2,28 @@
 ![dbt-core](https://img.shields.io/badge/dbt_core-version_>=1.0.0_<2.0.0-orange.svg)
 
 # GitHub dbt Package ([Docs](https://dbt-github.netlify.app/))
```

Comment:
We use the term “data model” instead of “dbt package” in the docs - what do you think about using it here too? Or is “dbt package” a common enough term that we should keep it?

Reply:
In my opinion we should keep dbt package. The people reading this will understand dbt package more and would become more confused if we start using data model instead. I think for marketing data model is good, but for our technical docs I think we should stick with package.

Reply:
Sounds good to me!